engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -53,19 +57,64 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
+    *      This should probably return a Try[BackendJobDescriptor], Unit, Boolean,
+    *      Try[ActorRef], or something to indicate if the job was started
+    *      successfully.  Or, if it can fail to start, some indication of why it
+    *      failed to start
+    */
+  private def startJob(jobKey: BackendJobDescriptorKey,
+                       inputs: Map[FullyQualifiedName, WdlValue],
+                       configDescriptor: BackendConfigurationDescriptor,
+                       factory: BackendLifecycleActorFactory): WorkflowExecutionActorState = {
+    val jobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, inputs)
+    val jobExecutionActor = context.actorOf(
+      factory.jobExecutionActorProps(
+        jobDescriptor,
+        BackendConfigurationDescriptor(configDescriptor.backendConfig, CromwellConfiguration.config)
+      ),
+      s"$workflowId-BackendExecutionActor-${jobDescriptor.key.tag}"
+    )
+    jobExecutionActor ! ExecuteJobCommand
+    WorkflowExecutionInProgressState
+  }
+
+  private def startJob(jobKey: BackendJobDescriptorKey, inputs: Map[FullyQualifiedName, WdlValue]): WorkflowExecutionActorState = {
+    workflowDescriptor.backendAssignments.get(jobKey.call) match {
+      case None =>
+        val message = s"Could not start call ${jobKey.tag} because it was not assigned a backend"
+        log.error(s"$tag $message")
+        context.parent ! WorkflowExecutionFailedResponse(Seq(new Exception(message)))
+        WorkflowExecutionFailedState
+      case Some(backendName) =>
+        val attemptedConfigurationDescriptor = BackendConfiguration.backendConfigurationDescriptor(backendName)
+        val attemptedActorFactory = CromwellBackend.shadowBackendLifecycleFactory(backendName)
+
+        def configurationDescriptorException(exception: Throwable) = new Exception(s"Could not get BackendConfigurationDescriptor for backend $backendName", exception)
+        def backendLifecycleActorException(exception: Throwable) = new Exception(s"Could not get BackendLifecycleActor for backend $backendName", exception)
+
+        (attemptedConfigurationDescriptor, attemptedActorFactory) match {
+          case (Success(configDescriptor), Success(factory)) =>
+            startJob(jobKey, inputs, configDescriptor, factory)
+          case (Failure(ex), Success(_)) =>
+            context.parent ! WorkflowExecutionFailedResponse(Seq(configurationDescriptorException(ex)))
+            WorkflowExecutionFailedState
+          case (Success(_), Failure(ex)) =>
+            context.parent ! WorkflowExecutionFailedResponse(Seq(backendLifecycleActorException(ex)))
+            WorkflowExecutionFailedState
+          case (Failure(ex1), Failure(ex2)) =>
+            context.parent ! WorkflowExecutionFailedResponse(Seq(configurationDescriptorException(ex1), backendLifecycleActorException(ex2)))
+            WorkflowExecutionFailedState
```

Comment:
I hear ya... I figured that since this has to be refactored anyway (see comment above about the return value), somebody can figure out how to be more clever.